pkg/reconciler/contour/config/contour.go

```diff
@@ -70,6 +83,37 @@ func NewContourFromConfigMap(configMap *corev1.ConfigMap) (*Contour, error) {
 		return nil, err
 	}
 
+	cors, ok := configMap.Data[corsPolicy]
+	if ok {
+		if err := yaml.Unmarshal([]byte(cors), &contourCORSPolicy); err != nil {
+			return nil, err
+		}
+
+		fields := [][]string{
+			contourCORSPolicy.AllowOrigin,
```

Comment:
The regexp check isn't applied to `AllowOrigin`

https://github.com/projectcontour/contour/blob/fe259fa4f9060210b4e43e49aab9a5daedf84c94/apis/projectcontour/v1/httpproxy.go#L505-L507

Reply:
👀